### ISSUE TITLE
Unable to build from Dockerfile

### ISSUE THREAD

**Issue by aj120426394** [docker/for-win#2444]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: FA91F626-98F3-4C29-B572-39C5E7399BA0/20180817092329

### Expected behavior
Run `docker build .` and it will build the image

### Actual behavior
Run `docker build .`
```
Sending build context to Docker daemon  24.06kB 
Step 1/2 : FROM wnameless/oracle-xe-11g:latest  
 ---> 698cc7361de4  
Step 2/2 : COPY test.txt /   
failed to copy files: invalid arguments   
```
### Information

  - Windows Version: Windows 10 Enterprise 1709 
  - Docker for Windows Version: 18.06.0-ce (windows container)
```
# /Dockerfile

FROM wnameless/oracle-xe-11g:latest
COPY test.txt /
```

This dockerfile is working on Mac or Linux container on Docker for Windows. However, my work machine has the customised version of Windows 10 which disable lots of features. This make the machine not able to share drive with Linux container on Docker for Windows. So the only way to mount my project code into docker container is using Windows container on Docker for Windows.

### Steps to reproduce the behavior

  1. `docker build .`

**Comment by kimjeff49** [comment]
Are you able to post your dockerfile by any chance?

**Comment by aj120426394** [comment]
@kimjeff49 My dockerfile is already there, it's pretty simple because it just a simple testing. This dockerfile is working on Mac or Linux container on Docker for Windows. However, my work machine has the customise version of Windows 10 which disable lots of feature. This make the machine not able to share drive with Linux container on Docker for Windows. 

So the only way to mount my project code into docker container is using Windows container on Docker for Windows. And here come the problem that I am not able to build the image with my Dockerfile.

**Comment by AlinaPenkina** [comment]
Hello, I have the same problem who could solve it? Tell me please

**Comment by alexey-y** [comment]
Same problem here...

Docker 18.06.1-ce-win73 (19507)
Windows 10 Professional  1803, 17134.228

Line 
ADD wowza-lib.tar /usr/local/WowzaStreamingEngine/lib/

is failing with

"Step 12/16 : ADD wowza-lib.tar /usr/local/WowzaStreamingEngine/lib/
failed to copy files: invalid arguments"

**Comment by aj120426394** [comment]
@AlinaPenkina  I finally figure out that my company disable a service on the Windows 10 called Server. And this service control the file sharing function on Windows 10. To check this, search "Service" on your windows, and find Server on the Service manager. If it shows disabled, right click on it and select properties. Change disabled to Automatically should solve the problem.

You might need super admin authorisation with your account to do this. Also, confirm with your security team see if they disable it automatically. In my case, our security team will scan each of the working machine daily and turn the service off. If that is your case, ask them to remove you from the scan list, or you need to turn it back manually.

**Comment by sworisbreathing** [comment]
@aj120426394 - I'm getting this issue but the Server service is running, so I think there's another problem going on here.

**Comment by Ofear** [comment]
Hi Guys
I had the same issue (Waste on it a whole day!)
The solution in my case was the switch to Linux containers
Seems like the Windows containers is not so stable.

Hope it help you guys

**Comment by augi** [comment]
If you are running LCOW and trying to build/run Linux containers then try to add `--platform=linux` explicitly. In LCOW, the Docker daemon handles both Linux and Windows containers and so maybe this is not fully implemented. See my relevant issue: https://github.com/docker/for-win/issues/2720

**Comment by augi** [comment]
I haven't a chance to try it so I don't know yet. I read about this workaround in another issue (but I lost the link).
Anyway, as I know, LCOW is distributed in stable release but as experimental feature (and there is a warning dialog when switching to Windows Containers).

**Comment by daviek19** [comment]
> 
> 
> > try to add `--platform=linux` explicitly.
> 
> So after adding --platform=linux COPY and ADD worked fine?
> 
> Actually, I don't see an added value for a windows container when dealing with Unix machines.
> I think that Windows containers need to return to BETA.
> COPY and ADD are pretty important in Docker.

This worked like a charm. Thank you.

**Comment by augi** [comment]
Maybe it's fixed 18.09 release because it contains this fix: https://github.com/moby/moby/pull/37563
